File: 1. Two Sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numsMap = {}
        for leftIdx, left in enumerate(nums):
            diff = target - left
            if (rightIdx := numsMap.get(diff)) is not None:
                return [leftIdx, rightIdx]
            numsMap[left] = leftIdx

File: 1. Two Sum/solution.php
<?php

class Solution
{

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer[]
     */
    function twoSum($nums, $target)
    {
        $numsMap = [];
        foreach ($nums as $leftIdx => $left) {
            $diff = $target - $left;
            if (($rightIdx = $numsMap[$diff] ?? null) !== null) {
                return [$leftIdx, $rightIdx];
            }
            $numsMap[$left] = $leftIdx;
        }
    }
}
